package.json

```diff
@@ -6,9 +6,9 @@
   "license": "Apache-2.0",
   "main": "src/index.js",
   "scripts": {
-    "ava": "ava test",
+    "ava": "ava -v test",
     "ci": "npm run lint && npm run coverage",
-    "coverage": "nyc ava test",
+    "coverage": "nyc ava -v test",
```

Comment:
I don't agree with this. If you want verbose tests, you can append it to the call, e.g. `npm run ava -- -v` or apparently `CI=true npm run coverage` (as Travis CI shows it)